go.mod

```diff
@@ -851,3 +851,5 @@ replace (
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.18.0
 	github.com/prometheus/common => github.com/prometheus/common v0.46.0
 )
+
+replace github.com/DataDog/agent-payload/v5 => github.com/DataDog/agent-payload/v5 v5.0.112-0.20240403100753-82d9646d460f
```

Comment:
note: requires https://github.com/DataDog/agent-payload/pull/297 to be merged before removing this